spring/spring-web/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>spring-web</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: spring-web</name>
```

Comment:
```suggestion
    <name>Quarkus QE TS: Spring: Spring Web</name>
```

Reply:
Thanks for pointing it out. I also fixed the name for previously migrated `spring/spring-data` module and added it to this PR.